Orion.Net/Authorization/PostApiVerificationHandler.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Orion.Net.Interfaces;
+
+namespace Orion.Net.Authorization
+{
+    public class PostApiVerificationHandler : AuthorizationHandler<PostApiVerification>
```

Comment:
Please rename PostApi and PostApiVerification by SupportID and SupportIDVerification